stylesheets/style.css

```diff
@@ -0,0 +1,122 @@
+
+@keyframes rainbow-text {
+    0% {color:red;}
+    12.5% {color:orange;}
+    25% {color:gold;}
+    37.5% {color:limegreen}
+    50% {color:cornflowerblue;}
+    62.5% {color:indigo;}
+    75% {color:purple;}
+    87.5% {color:magenta;}
+    100% {color: red;}
+}
+
+body {
+    background-color: darksalmon;
+    font-family:'Varela Round', Verdana, Geneva, Tahoma, sans-serif;
+    line-height: 1.5;
+    font-size: 12pt;
+    color: darkred;
+    margin: 0;
+    padding: 0;
+}
+
+article, .title-text {
+    background: wheat;
+    background: linear-gradient(180deg, wheat, lemonchiffon);
```

Comment:
There are a few places in the CSS where `background` is listed twice. Make sure to keep the code DRY!